### DISCUSSION TITLE
Arch Linux

### DISCUSSION THREAD

**Discussion by ikalnytskyi** [shopokodu/community#1] [Теми для подкасту]
В YouTube чаті під час етеру наживо, неодноразово питали зробити випуск про Arch Linux.

**Comment by rkiyanchuk** [comment]
## Що особливого в Arch Linux?

- гнучкість (можна вибрати будь-яку комбінацію ПЗ – дистрибутив не диктує чим користуватись)
- як наслідок: відсутній інсталятор. Систему потрібно встановлювати повністю руками.
- простота збірка пакетів
- прагматизм (запаковані зокрема і пропрієтарні бінарі).

## План обговорення

1. Як ви прийшли до Linux взагалі, і конкретно до дистрибутиву Arch?
2. Чому надаєте перевагу саме Arch?
    - Руслан: rolling release, Arch Wiki
3. Який у вас підхід до вставновлення Arch?
    - Руслан: [Ansible playbook](https://github.com/rkiyanchuk/ansible-arch-linux) з повним bootstrap.
4. Основні проблеми що у вас виникали з Arch?
    - Чи були проблеми через rolling release оновлення?
    - Підтримка hardware або відсутність драйверів?
5. Яким AUR-helper користуєтеся?
    - Руслан: [yay](https://github.com/Jguer/yay)
6. Яке у вас оточення?
    - Руслан: Xorg,XMonad ❤️ -> Wayland/Sway/Pipewire 💩

7. Шо по підтримці ARM? А Risc-V?

## Бонус

* Новий дистрибутив на базі Arch: [Crystal](https://getcryst.al/site).
